src/components/Carousel/CarouselContent.ts

```diff
@@ -17,4 +17,6 @@ export type CarouselContent = UniBlockContent & {
   isCyclic?: boolean;
   /** @title Внутренний отступ  */
   isPadding?: boolean;
+  /** @title Выравнивание по центру  */
```

Comment:
```suggestion
  /** @title Выравнивать по центру  */
```
Так, наверное, получше будет